model/src/main/java/org/projectnessie/api/v2/http/HttpTreeApi.java

```diff
@@ -176,8 +176,11 @@ SingleReferenceResponse createReference(
         content = {
           @Content(
               mediaType = MediaType.APPLICATION_JSON,
-              examples = {@ExampleObject(ref = "refObj")},
-              schema = @Schema(implementation = Reference.class))
+              examples = {
+                @ExampleObject(ref = "singleReferencesResponse"),
```

Comment:
Quarkus swagger UI error (in browser): 
```
Resolver error at paths./api/v2/trees/{ref}.get.responses.200.content.application/json.examples.singleReferencesResponse.$ref
Could not resolve reference: Could not resolve pointer: /components/examples/singleReferencesResponse does not exist in document
```

It looks like a typo, it should probably be `singleReferenceResponse` (not plural, as in openapi.yaml).